src/10_functions.py

```diff
@@ -9,4 +9,9 @@
 # Print out "Even!" if the number is even. Otherwise print "Odd"
 
 # YOUR CODE HERE
+def is_even(num):
+	if int(num) % 2 == 0:
+		print("Even!")
+	else:
+		print('Odd!')
 
```

Comment:
Great work here! We can save 3 lines of code by simply writing return n % 2 == 0 which will return a Boolean in it of itself. Honestly though, sometimes I find the way you did it to be more readable. However I thought I'd mention it so you are aware if you were not already!